### PR TITLE
Create logo.png

### PR DESCRIPTION
**Pull request by WutHonker**
logo

### CODE REVIEW

File: bsc/assets/0x46ef6cdf5174fb92fa92c0dfcf55b2ff5ee4f8f9/logo.png [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (WutHonker) [inline bsc/assets/0x46ef6cdf5174fb92fa92c0dfcf55b2ff5ee4f8f9/logo.png:1]: logo

**Comment by WutHonker** [comment]
@lukebogtools

**Comment by lukebogtools** [comment]
Hey, please change the folder name to the CHECKSUM'd contract address (the one with some capitalisation). Otherwise your token info wont display. You can get the checksum'd address by pasting your contract address here: https://ethsum.netlify.app/

Cheers,
Luke

**Comment by lukebogtools** [comment]
Please reopen when you've made the change. Ty :)

**Pull request closed without merging**